test/spec/todo/todo.spec.ts

```diff
@@ -80,4 +80,79 @@ describe("DELETE /todos/:id", () => {
     const res3 = await chai.request(expressApp).delete(`/todos/""`);
     expect(res3).to.have.status(500);
   });
+  describe("PUT /todos/:id", () => {
+    it("should return 200 if todo exists and title is validately true and 400 if title is empty or not a string else 404", async () => {
```

Comment:
This is pretty bad test case.

A test case should be granular and should test only scenario at a time.

Reply:
okay